ra_aid/webui/server.py

```
@@ -0,0 +1,220 @@
+                        })
+                        
+                    except Exception as e:
+                        logger.error(f"Error executing ra-aid: {e}")
+                        await manager.handle_error(websocket, str(e))
+                        
+            except Exception as e:
+                logger.error(f"Error processing message: {e}")
+                await manager.handle_error(websocket, str(e))
+                
+    except WebSocketDisconnect:
+        logger.info(f"WebSocket client {client_id} disconnected")
```

Ruff (0.8.2)

Logging statement uses f-string

(G004)